### DISCUSSION TITLE
Quick Panel is not filling evenly

### DISCUSSION THREAD

**Discussion by JacobCoffee** [zellij-org/zellij#2832] [Q&A] [answered]
<details>
<summary>Current Behavior Image</summary>

<img width="860" alt="image" src="https://github.com/zellij-org/zellij/assets/45884264/bbd21c55-6208-4ecd-bb68-d93dedf97939">

</details>

Sorry if I am missing something, but when I quickly run some new panels with `alt+n` it seems to all fill on the right side by default. I could've sword it didnt do this some versions ago, but maybe that's a me thing... I know that `ctrl+p -> d` will add the panel down but I think it would make sense to fill evenly when I hit `alt+n` like so:

<details>
<summary>Expected Behavior Image</summary>

<img width="862" alt="image" src="https://github.com/zellij-org/zellij/assets/45884264/9b2663f2-d6df-483d-acc6-eeb936dfd493">

</details>

Is there some config/template/setting I am missing to do this?

**Comment by JacobCoffee** [comment]
Also, as an aside I read somewhere that we currently can't hide/remove the tip/action bar sections
<img width="846" alt="image" src="https://github.com/zellij-org/zellij/assets/45884264/a5f60392-674b-4569-9dfa-34f9aaada20f">

Is this still true? re (sortve): https://github.com/zellij-org/zellij/discussions/1270

it is quite bulky for when I start remembering the shortcuts, I dont really need the extra space taken from me.

**Comment by imsnif** [comment, marked as answer]
Hey, for the first issue, you might want to disable auto layout: https://zellij.dev/documentation/options.html?highlight=auto_l#auto_layout (either that or create a [swap layout](https://zellij.dev/documentation/swap-layouts) that works for you, you can toggle the existing default ones with `Alt` + `[]`).

For the second issue, check out the compact bar: `zellij -l compact`.

**Reply by JacobCoffee** [reply]
doh... these worked. On the second one i only disabled pane frames. This worked! tyvm
